src/Admin/AbstractAdmin.php

```diff
@@ -3533,6 +3535,8 @@ protected function configureDefaultFilterValues(array &$filterValues)
      * Example:
      *   $sortValues['_sort_by'] = 'foo'
      *   $sortValues['_sort_order'] = 'DESC'
+     *
+     * @phpstan-param array{_page?: int, _per_page?: int, _sort_by?: string, _sort_order?: string} $sortValues
```

Comment:
this could be simply param, afaik array shape is supported by Ides like phpstorm right @VincentLanglet

Reply:
Phpstorm has a weird support about this.

![image](https://user-images.githubusercontent.com/9052536/91669709-46a1b780-eb17-11ea-8257-6705c4377559.png)

It does understand array shape for return type.
But not fully for param type. (You can see that `$foo` is considered as a comment).

`@phpstan-param` seems to be more consistent with our new standard.

Reply:
AFAIK this is no standard doc, so we should keep both annotations.

Reply:
The [RFC](https://github.com/php-fig/fig-standards/blob/master/proposed/phpdoc.md#arrays) does not mention this complex array information